jobs/search-alert/search_alert/main.py

```diff
@@ -0,0 +1,337 @@
+import click
+import datetime as dt
+import holidays
+import numpy as np
+import pandas as pd
+from google.cloud import bigquery
+
+
+# check whether the day +/- 1 day is holiday
+def is_it_holiday(ds, country):
+	isHoliday = False
+	ds_range = [pd.to_datetime(ds) + dt.timedelta(days = x) for x in range(-2,2,1)] # get clever about the holiday if it's close to weekend
+	try:
+		isHoliday_range = [pd.to_datetime(ds) in holidays.CountryHoliday(country) for ds in ds_range]
+		isHoliday = max(isHoliday_range)
+	except:
+		pass
```

Comment:
except pass? What exception do you expect to see here?

Reply:
I'm not sure, this script is a notebook I'm converting, I'd rather leaver it as is.

Reply:
ah ok, in that case let's leave it be